packages/perseus-editor/src/components/graph-type-selector.tsx

```diff
@@ -7,25 +9,34 @@ type GraphTypeSelectorProps = {
 
 const GraphTypeSelector = (props: GraphTypeSelectorProps) => {
     return (
-        <select
-            value={props.graphType}
-            onChange={(e) => {
-                const type = e.target.value;
-                props.onChange(type);
+        <SingleSelect
+            selectedValue={props.graphType}
+            onChange={(newValue) => {
+                props.onChange(newValue);
             }}
```

Comment:
I'm ok either way, but we can shorten this up if we want:
```suggestion
            onChange={props.onChange}
```